doc/source/configuration.rst

```diff
@@ -3,115 +3,258 @@
 solutions first, as the necessary effort can vary significantly depending on your specific needs.
 
+.. _configuration-initial-setup:
+
+Create the inital database setup
+--------------------------------
+
+Out of the box the pyramid_oereb server supports three different topic configurations in the database:
+
+  - the **pyramid_oereb standard model**
+  - the **interlis 2.3 OeREBKRM transfer model**
+  - the **OEREBlex topic model**
+
+Pyramid_oereb Standard Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This schema and table structure is based on the initial topic structure used in the pyramid_oereb
+v1.x versions. It's mainly used for cantonal topics which are not (yet) stored in the interlis 2.3 OeREBKRM-
+Transfer structure and by cantons that do not use OeREB-Lex to manage the legal documents.
+
+Interlis 2.3 OeREBKRM Transfer Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All the federal data sets are provided in this data structure. So this is the schema and table model you
+want to use for all the federal topics unless you want to remap the data to a specific database structure.
+If your cantonal data is also stored based on this model, then you probably want to use this structure 
+for all topics to homogenize your database content.
+The `Ili2pg Oereb Data Import Manual <https://github.com/openoereb/ili2pg_oereb_data_import_manual>`__
+explains how to use ili2pg tool to create the corresponding schema and how to import the XML data.
+
+
+OEREBlex Topic Model
+^^^^^^^^^^^^^^^^^^^^
+
+This third model is usefull if you maintain your legal documents using the OEREBlex application and you
+have a specific cantonal model for your data. It is similar to the pyramid_oereb standard model, but all
+the document related tables are omitted. Instead the documents are linked with the geolink attribute. 
 
 .. _configuration-additional-topics:
 
-Add additional topics
----------------------
+Add additional standard topics
+------------------------------
+
+If you like to add one or more additional topics based on the *pyramid_oereb standard* database structure
+you can use the internal command below creating an SQL script to establish the topic schema.
+
+But before creating any new topic structure you have to add its configuration. Open the configuration file
+(pyramid_oereb.yml) and copy the section from one of the existing **standard** topics which usually 
+looks like this:
+
+.. code-block:: yaml
 
-If you only need to add one or more additional cantonal topics and want to use the standard database structure
-for these too, you can run a provided script to generate the required models.
+    - code: ch.NE.Baulinien
+      geometry_type: LINESTRING
+      thresholds:
+        length:
+          limit: 1.0
+          unit: 'm'
+          precision: 2
+        area:
+          limit: 1.0
+          unit: 'm²'
+          precision: 2
+        percentage:
+          precision: 1
+      language: fr
+      federal: false
```

Comment:
In which context is the parameter "federal" still relevant/used?

Reply:
@michmuel it is still in the pyramid_oereb.yml.mako at least...

Reply:
The federal flag is still being used to define if a topic should be [listed or skipped ](https://github.com/openoereb/pyramid_oereb/blob/b900ffa23b1392b60d16a9f64d0504deab8ba252/pyramid_oereb/core/views/webservice.py#LL809C34-L809C34)for the extract.
But I don't know the use-case for that to be honest.

Reply:
I reworked the documentation. May I ask you to revise this part and tell me how to proceed?

Reply:
@voisardf @michmuel @svamaa : I will resolve this comment and merge the PR. As far as I understand this is still used.